RHEL/6/templates/csv/sysctl_values.csv

```diff
@@ -19,3 +20,9 @@ net.ipv4.ip_forward,0
 net.ipv4.tcp_syncookies,
 net.ipv6.conf.default.accept_ra,
 net.ipv6.conf.default.accept_redirects,
+net.ipv6.conf.all.accept_ra,#only-for:oval
+net.ipv6.conf.all.accept_redirects,#only-for:oval
+#net.ipv6.conf.all.disable_ipv6,1
```

Comment:
Why have you added this and commented it out? I can't find any such rule in current master.